internal/platform/open_file_windows.go

```diff
@@ -140,6 +142,16 @@ func open(path string, mode int, perm uint32) (fd syscall.Handle, err error) {
 			}
 		}
 	}
+
+	if strings.Contains(runtime.Version(), "go1.20") {
```

Comment:
maybe cache this as a var since it won't change?

Reply:
will do after CI went green